src/sys/socket/addr.rs

```diff
@@ -217,6 +217,15 @@ impl AddressFamily {
             libc::AF_NETLINK => Some(AddressFamily::Netlink),
             #[cfg(any(target_os = "macos", target_os = "macos"))]
             libc::AF_SYSTEM => Some(AddressFamily::System),
+            #[cfg(any(target_os = "android", target_os = "linux"))]
```

Comment:
Because of this addition the doccomment for `from_i32` needs to be updated.

Reply:
so ?

Reply:
So what?

Reply:
as your wish, document has updated.